cli/cmd/report/report.go

```diff
@@ -56,3 +55,19 @@ func setCmdByOS() (string, []string, error) {
 		return "", nil, errors.New("it wasn't possible to identify your OS")
 	}
 }
+
+func isWSL() (*bool, error) {
```

Comment:
I chose a boolean reference because in a error case this output can be nil since I think it is more clear than true or false.

Reply:
I see! I think we should remove the reference because it is breaking a bit away from common patterns. I think a reference to a more complex type would be OK, but for a primitive type like `bool`, we want to use the "zero value" instead.

All callers of a function that return an `error` will always check the error value before using the result :smile:

Reply:
cool, I will change it then